### DISCUSSION TITLE
MIDIBLE Receive and Forward to Serial+USB

### DISCUSSION THREAD

**Discussion by raviumadi** [pschatzmann/arduino-stk#12] [Q&A] [answered]
Hi Phil,
I am trying to figure this out: I am sending MIDI messages using ble-send.ino, this works great when connected to Garageband on iOS/MacOS. However, I would like to build another system to recieve the messages and forward to a MIDI USB device. I tried the ble-receive.ino but nothing seems to be happening when I connect it to iOS/MacOS and hit piano keys on Garageband. I was expecting to see messages on serial monitor in Arduino IDE but noting happens. Also, how do I pair the ble-receive module with ble-send device?
Thank you :-)

**Comment by pschatzmann** [comment, marked as answer]
I don't see any connection with STK: This is an audio processing framework and not a midi library!

**Comment by raviumadi** [comment]
Ah my bad. I was looking at your Arduino Midi library as well and posted on the wrong discussion page.
